assets/src/edit-story/components/popup/index.js

```diff
@@ -56,26 +51,26 @@ export const Placement = {
 };
 
 const Container = styled.div.attrs(
-  ({ x, y, width, height, fillWidth, fillHeight }) => ({
+  ({ x, y, width, height, fillWidth, fillHeight, placement }) => ({
     style: {
-      left: `${x}px`,
-      top: `${y}px`,
+      transform: `translate(${x}px, ${y}px) ${getTransforms(placement)}`,
       ...(fillWidth ? { width: `${width}px` } : {}),
       ...(fillHeight ? { height: `${height}px` } : {}),
     },
   })
 )`
+  left: 0px;
+  top: 0px;
```

Comment:
this allows the content to render at its proper size, before moving it to the proper position.